apps/clock/js/picker/picker.js

```diff
@@ -44,22 +39,18 @@ define(function(require) {
       var values = [];
       var range = setup.pickers[picker].range;
       var isPadded = setup.pickers[picker].isPadded || false;
-      var valueText = setup.pickers[picker].valueText;
-      var textValues = [];
 
       this.nodes[picker] = setup.element.querySelector('.picker-' + picker);
 
       for (var i = range[0]; i <= range[1]; i++) {
-        values.push(isPadded && i < 10 ? '0' + i : '' + i);
-        if (valueText) {
-          textValues.push(_(valueText, { n: i }));
-        }
+        values.push(i);
       }
 
       this.spinners[picker] = new Spinner({
         element: this.nodes[picker],
         values: values,
-        textValues: textValues.length ? textValues : values
+        isPadded: isPadded,
+        l10nId: setup.pickers[picker].l10nId
       });
```

Comment:
At this point we're in a good shape to follow the spec as we finally have a near-final, polished one.
> [in RTL] Continuous/Spinner Icon and animation (should point and move counter clockwise)

From Page 4
https://bug1179459.bmoattachments.org/attachment.cgi?id=8663887
So indeed we should mirror those :)